portfolio/src/main/webapp/script.js

```diff
@@ -30,8 +30,8 @@ function addRandomLanguage() {
 /**
  * Appends previously made comments that populated the servlet as children list elements to the page.
  */
-function showComments() {
-  fetch('/data').then(response => response.json()).then((commentsJson) => {
+function showComments(val) {
```

Comment:
Can you rename 'val' so it's more descriptive?

Reply:
Done!